modules/nf-core/emu/abundance/tests/main.nf.test.snap

```diff
@@ -6,26 +6,23 @@
                     {
                         "id": "test"
                     },
-                    [
-                        "test.fastq_read-assignment-distributions.tsv:md5,c5c52bef375dee47407b3711b147b61d",
-                        "test.fastq_rel-abundance.tsv:md5,6fb86cb4103ae5064bb2b7b43e9c9c24"
-                    ]
+                    "test_rel-abundance.tsv:md5,6fb86cb4103ae5064bb2b7b43e9c9c24"
                 ]
             ],
             [
                 [
                     {
                         "id": "test"
                     },
-                    "test.fastq_read-assignment-distributions.tsv:md5,c5c52bef375dee47407b3711b147b61d"
+                    "test_read-assignment-distributions.tsv:md5,c5c52bef375dee47407b3711b147b61d"
                 ]
             ],
             [
                 [
                     {
                         "id": "test"
                     },
-                    "test.fastq_emu_alignments.sam:md5,09f832656b47f44ff22f65ff7ac87824"
+                    "test_emu_alignments.sam:md5,55a142107da27ea7d9e683af2a5f262f"
```

Comment:
They are still present in the output in a normal run, so they should also be present in the stub test 😉